### PR TITLE
Generalize ISO offset instructions

### PR DESCRIPTION
**Pull request by andrewdavidwong**
This is a user-requested change.

### CODE REVIEW

File: user/troubleshooting/uefi-troubleshooting.md [hunk 1]
```diff
@@ -21,10 +21,11 @@ If you've installed successfully in legacy mode but had to change some kernel pa
 
 **Change xen configuration directly in an iso image**
 01. Get EFI partition boundaries `parted Qubes-R4.0-rc4-x86_64.iso unit B print`
-02. Using the start address and the size of the EFI partition, setup a loop device for it `sudo losetup -o 524288 --sizelimit 30562304 /dev/loop0 Qubes-R4.0-rc4-x86_64.iso`
-03. Mount the loop device `sudo mount /dev/loop0 /mnt`
-04. Edit `EFI/BOOT/BOOTX64.cfg` to add your params to the `kernel` configuration key
-05. Save your changes, unmount and dd to usb device
+02. Find the offset for your ISO (see [here](https://unix.stackexchange.com/questions/270171/why-offset-specific-in-loop-mount)).
+03. Using the start address and the size of the EFI partition, setup a loop device for it. This depends on the offset you found in the previous step and is specific to your ISO. Here is an example command using a specific Qubes ISO (if your ISO is different, you will have to adjust the command accordingly): `sudo losetup -o 524288 --sizelimit 30562304 /dev/loop0 Qubes-R4.0-rc4-x86_64.iso`
```
Comment (marmarek) [inline user/troubleshooting/uefi-troubleshooting.md:25]: In fact, it should be much simpler to just use `losetup -P` and let the kernel parse the partition table. This way, you'll get `/dev/loop0p2` with the EFI partition. No need to manually entering any offsets.

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by andrewdavidwong** [comment]
I don't understand why the CI check failed. I'll retry it.

File: user/troubleshooting/uefi-troubleshooting.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ If you've installed successfully in legacy mode but had to change some kernel pa
 04. Install using your modified boot entry
 
 **Change xen configuration directly in an iso image**
-01. Get EFI partition boundaries `parted Qubes-R4.0-rc4-x86_64.iso unit B print`
-02. Using the start address and the size of the EFI partition, setup a loop device for it `sudo losetup -o 524288 --sizelimit 30562304 /dev/loop0 Qubes-R4.0-rc4-x86_64.iso`
-03. Mount the loop device `sudo mount /dev/loop0 /mnt`
+01. Get EFI partition boundaries (replacing `X` with your ISO's version name): `parted Qubes-RX-x86_64.iso unit B print`
```
Comment (marmarek) [inline user/troubleshooting/uefi-troubleshooting.md:23]: This step is unnecessary in the new version.

File: user/troubleshooting/uefi-troubleshooting.md [hunk 3]
```diff
@@ -20,9 +20,9 @@ If you've installed successfully in legacy mode but had to change some kernel pa
...
+02. Set up a loop device (replacing `X` with your ISO's version name): `losetup -P /dev/loop0 Qubes-RX-x86_64.iso`
+03. Mount the loop device: `sudo mount /dev/loop0 /mnt`
```
Comment (marmarek) [inline user/troubleshooting/uefi-troubleshooting.md:25]:
```suggestion
03. Mount the loop device: `sudo mount /dev/loop0p2 /mnt`
```

**Review by marmarek — CHANGES_REQUESTED** [review]

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of fcc3f650d565f3a6aa79aa593df175854a62039e
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmBP3Z8ACgkQ8yiUvpaE
k4qjMw/+MQ4rW5WFROMnz+6/VovbcDZz6X5cNFDIwbJGLg064bRUDhNnr2w2If/J
SBmV3TOtzwNgt6FajCOaHSvhsFaR2O/jlOUu6OGPycd0ZbzgjtpXP9Ri9CtF9MIJ
amHqfxYs5CxSDPy8tn/b5p9CIXiddZujpxZVPcW+5UPGu7WNSvQamLlKXjEfh1hS
V1/LXPDKBNsGEp80nCuVmGcxn2U6+h/arMQLCzb7m6aFUjHX855sQCAONHET2bsQ
paqah7VIgGm9XJJmXu4BHSBOuI48NMxih1bX4bM1iLwwUjXCEVjmcUJcw7OBNQ89
5AwjTjcIKRiuP3b6UTrZBTxOxTIfKZuoLM5EDDY/SLd++IsWcRdhJBnUzMY4ljCc
BoZuaKNB3y9JVdk/F/krmien5Ed7yO7t0zgioXgr0ZUxXLad4gtVo2YqBT9gpy+r
ASWIsnGGcfIw+Bbhqc8IqmsWNeMw4NTWu8OG48sGq7VECehYenCba3DJssQmUyg4
9SBfWEBwgKS4ySdxC4kM8P/C0laF+N3TwZ0Hmz2a9ONhZmXBVEjBxLK0ul55O8pH
5FUDl7YsiLagT6fZooIazWyvz1WQ0Zs7Bx1suHUZ2UyBqBDSJMPWTUzhj7GmlxgR
ZwWpwBozoK9mMgCZy4X1FiwbqTpGEUNzXXCuG85M9xmuYUfvGtY=
=/QSr
-----END PGP SIGNATURE-----
```

**Pull request merged**